programs/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+set(programs_target "${MBEDTLS_TARGET_PREFIX}programs")
+add_custom_target(${programs_target})
+
+set(ssl_opt_target "${MBEDTLS_TARGET_PREFIX}ssl-opt")
```

Comment:
`ssl-opt` should also generate `tls13-compat.sh` which is added by https://github.com/Mbed-TLS/mbedtls/pull/9563. This can only be done once both parts of the code are in — reminder to make a follow-up PR.